universal-login-wallet/src/services/Notifications.ts

```diff
@@ -0,0 +1,33 @@
+import {Procedure, ETHER_NATIVE_TOKEN} from '@universal-login/commons';
+import WalletService from './WalletService';
+import UniversalLoginSDK from '@universal-login/sdk';
+import {transactionDetails} from '../config/TransactionDetails';
+
+export default class NotificationsService {
+  notifications = [];
+
+  constructor (private sdk: UniversalLoginSDK, private walletService: WalletService) {
+  }
+
+  subscribe (callback: Procedure) {
+    callback(this.notifications);
+    const subscription = this.sdk.subscribe(
+      'AuthorisationsChanged',
+      {contractAddress: this.walletService.userWallet!.contractAddress},
```

Comment:
extract variable
`const contractAddress = this.walletService.userWallet!.contractAddress;`
`{contractAddress, ...}`